crates/extensions/az_mapper_ext/src/converter.rs

```diff
@@ -48,11 +48,16 @@ impl AzureConverter {
     }
 
     fn try_convert(&mut self, input: &MqttMessage) -> Result<Vec<MqttMessage>, ConversionError> {
+        let default_timestamp = self.add_timestamp.then(|| self.clock.now());
         if is_bridge_health(&input.topic.name) {
             Ok(vec![])
+        } else if input.topic.name.starts_with("tedge/health") {
+            Ok(vec![
+                (MqttMessage::new(&self.mapper_config.out_topic, input.payload_str()?)),
```

Comment:
As the point is to only change the topic, I would avoid to touch the payload.

Reply:
refactored to just update the topic in the same payload.